.github/actions/api/breaking-change/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+set -e
+
+readonly base="$1"
+readonly revision="$2"
+readonly fail_on_diff="$3"
+readonly include_checks="$4"
+echo "running oasdiff breaking base: $base, revision: $revision, fail_on_diff: $fail_on_diff, include_checks: $include_checks"
+
+go install github.com/treeverse/oasdiff@v1.8.2
+# Build flags to pass in command
+flags=""
+if [ "$fail_on_diff" = "true" ]; then
+    flags="${flags} --fail-on WARN"
+fi
+if [ -n "$include_checks" ]; then
```

Comment:
This is odd.  We have one boolean-ish arg fail_on_diff that takes a boolean string and one boolean-ish arg include_checks that takes a nonempty string.  Consider e.g. passing the last arg as "false" or the third arg "1".

Perhaps just use standard sh flag parsing techniques?